packages/rule-components/src/ReportDetails/index.scss

```diff
@@ -20,9 +22,36 @@
     display: block;
     margin-top: var(--pf-l-stack--m-gutter--MarginBottom);
   }
+
+  .ins-c-rule__risk-description .battery .label {
```

Comment:
This is not a good way how to write SCSS, also shouldn't that class `.label` be `label` element?
```SCSS
.ins-c-rule__risk-description {
  .ins-c-rule__battery label { font-weight: var(--pf-global--FontWeight--bold); } 
  .ins-c-rule__severity-level-1 label { color: $ins-color--gray; }
  .ins-c-rule__severity-level-2 label { color: $ins-color--medium; }
  .ins-c-rule__severity-level-3 label { color: $ins-color--high; }
  .ins-c-rule__severity-level-4 label { color: $ins-color--critical; }
}
```

Reply:
No, it's `span` element with className `label`. Fixed as well